.github/workflows/goalie-on-pr-review-or-response.yaml

```diff
@@ -0,0 +1,55 @@
+# on a review from someone in the reviewers group, remove the awaiting-review label and add the awaiting-author label
+name: Update labels for awaiting review
+on:
+  pull_request_review:
+    types: [submitted]
+  pull_request_review_comment:
+    types: [created]
+
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v5
+        id: get-reviewers
+        with:
+          github-token: ${{ secrets.GH_SERVICE_ACCOUNT_TOKEN }}
+          script: |
+            const {data: reviewerMembers} = await github.rest.teams.listMembersInOrg({
+              org: context.payload.organization.login,
+              team_slug: 'reviewers'
+            });
+            return reviewerMembers;
+      - uses: actions/github-script@v5
+        id: fix-labels
+        with:
+          script: |
+            const isReviewer = ${{steps.get-reviewers.outputs.result}}.some(({login}) => login === context.actor);
```

Comment:
gh actions don't have the problem of treating the `-` in `get-reviewers` as a math minus, do they? :P

Reply:
Nope works perfectly :joy: